packages/ui/src/components/w3i-widget/index.ts

```diff
@@ -142,9 +101,6 @@ export class W3iWidget extends LitElement {
 
     return html`
       <div ${ref(this.iframeRef)} class="w3i-widget">
-        <button class="w3i-close-button" @click=${this.onCloseClick}>
-          <div class="w3i-close-button-foreground"></div>
-        </button>
```

Comment:
I know this was supposed to be replaced by a listener that checks for clicks outside the widget, but I don't see that here?

Reply:
Never heard of that requirement. Is there an issue for it ?
Anyway, that can be handled by the dapp that implements the widget thanks to your new hooks.
Also we can close the widget by clicking the bell and that X button isn't on any design so better remove it.

Reply:
Ya this is solved easily with the hooks, lgtm